### PR TITLE
provide gender field

### PR DESCRIPTION
**Pull request by hirschrobert**
tried to contribute to my issue #108 . If you could have a look at it. am new here and hope it is the right track. many thanks

### CODE REVIEW

**Comment by MariusBluem** [comment]
Well ... Do you have some example clients who render the gender in your vCard? 

Especially the non-male/female part 🤔 ... I think we should replace this with something like "other" or whatever those CardDAV-clients do ;)

**Comment by hirschrobert** [comment]
when i download it in nextcloud it gives this file: [a94d959f-a9e9-4d18-b887-4664370e4eda-4.vcf.txt](https://github.com/nextcloud/contacts/files/998564/a94d959f-a9e9-4d18-b887-4664370e4eda-4.vcf.txt)

According to https://tools.ietf.org/html/rfc6350#section-6.2.7 that seem to be right.

File: l10n/de_DE.js [hunk 1]
```diff
@@ -70,6 +70,10 @@ OC.L10N.register(
     "Voice" : "Anruf",
     "Social network" : "Soziales Netzwerk",
     "Details" : "Details",
-    "Settings" : "Einstellungen"
+    "Settings" : "Einstellungen",
+    "Gender" : "Geschlecht",
+    "Male" : "männlich",
+    "Female" : "weiblich",
+    "intersex" : "intersexuell"
```
Comment (MariusBluem) [inline l10n/de_DE.js]: Translations are handled over Transifex 🙈

Comment (hirschrobert) [inline l10n/de_DE.js]: thanks. do i understand [this](https://docs.nextcloud.com/server/11/developer_manual/core/translation.html#automated-synchronization-of-translations) right? I don't need to take care of translation? Your server is doing the scanning of new labels, pushes it to transifex, your crew will translate it and pushes it to github? All automatically? Thanks!

Comment (irgendwie) [inline l10n/de_DE.js]: @hirschrobert That's correct - you just have to add "t('contacts', ...)" in the code and the rest is handled automatically.

**Comment by skjnldsv** [comment]
No, don't delete the files, delete the edit from the commit :)

**Comment by xh3n1** [comment]
Female
Male
None
Other
Unknown

**Comment by MariusBluem** [comment]
Male
Female
Other

None/Unknown can be done by simply not adding this gender-field.

**Comment by jonatoni** [comment]
thanks @nickvergessen 😜

**Comment by MariusBluem** [comment]
Thanks for contributing 🙈

**Pull request merged**